clean: Clarify the repository configuration permission DOCS-658 (#2032)
codacy/docs: docs/organizations/roles-and-permissions-for-organizations.md

      <td class="yes">Yes</td>
    </tr>
    <tr>
      <td>Configure repository Git provider integration settings</td>
      <td class="no">No</td>
      <td colspan="2" class="no">No</td>
      <td colspan="2" class="no">No</td>
      <td class="yes">Yes</td>
      <td class="maybe">Inherits original permission</td>
      <td class="yes">Yes</td>
    </tr>
    <tr>
      <td>Configure repository quality gates and goals</td>
      <td class="no">No</td>
      <td colspan="2" class="no">No</td>

>       <td colspan="2" class="no">No</td>

Reviewer: ```suggestion
      <td colspan="2" class="yes">Yes</td>
```

      <td class="yes">Yes</td>
      <td class="maybe">Inherits original permission</td>
      <td class="yes">Yes</td>
    </tr>
    <tr>

>       <td>Enable repository local analysis, manage repository API tokens</td>

Author: @nicklem I incorporated your feedback for the new operations description.

>       <td>Configure repository to run analysis on local build server, manage repository API tokens</td>

Reviewer 2: Now that I'm reading it, I'm not convinced anymore 😅
We can either revert to your initial suggestion or reword this slightly. Up to you.

```suggestion
      <td>Set repository analysis to run on a local build server, manage repository API tokens</td>
```

Author: Using **Enable** instead, to match the description on the UI.

```suggestion
      <td>Enable repository analysis to run on a local build server, manage repository API tokens</td>
```

      <td colspan="2" class="yes">Yes</td>
    </tr>
    <tr>
      <td>Configure repository Git provider integration settings</td>
      <td class="no">No</td>
      <td colspan="2" class="no">No</td>
      <td class="no">No</td>
      <td colspan="2" class="yes">Yes</td>
      <td class="maybe">Inherits original permission</td>
      <td colspan="2" class="yes">Yes</td>
    </tr>
    <tr>
      <td>Configure repository quality gates and goals</td>
      <td class="no">No</td>
      <td colspan="2" class="no">No</td>

>       <td class="no">No</td>

Reviewer: ```suggestion
<td class="no">No</td>
```

>       <td class="no">No</td>

Reviewer: ```suggestion
      <td class="yes">Yes</td>
```
